File: C/FizzBuzz.c
#include <stdio.h>

#define MAX_VALUE 100;

int main (void)
{
    int i;

    for (i = 0; i < MAX_VALUE; i++)
    {
        if ((i % 15) == 0)
        {
            printf ("FizzBuzz");
        }
        else if ((i % 5) == 0)
        {
            printf ("Buzz");
        }
        else if ()
        {
            printf ("Fizz");
        }
        else
        {
            printf ("%d", i);
        }
    }
}

File: Java/FizzBuzz.java
public class FizzBuzz {
static final int MAX_VALUE = 100;

public static void main(String args[]) {
    for (int i = 0; i < MAX_VALUE; i++) {
        if ((i % 15) == 0) {
            System.out.println("FizzBuzz");
        } else if ((i % 5) == 0) {
            System.out.println("Buzz");
        } else if ((i % 3) == 0) {
            System.out.println("Fizz");
        } else {
            System.out.println(i);
        }
    }
}
}

